Tests/TestRx/FixtureRxVars.swift

```diff
@@ -1,708 +1,557 @@
-    }
-    private(set) var routeToFooCallCount = 0
-    var routeToFooHandler: (() -> Observable<()>)?
-    func routeToFoo() -> Observable<()> {
-        routeToFooCallCount += 1
-        if let routeToFooHandler = routeToFooHandler {
-            return routeToFooHandler()
+    @Fixture
+    enum parent {
+        public class BarMock: Bar {
+            public init() {}
         }
-        return Observable<()>.empty()
     }
-}
-
-"""
-
-
-let rxObservables = """
-/// @mockable(rx: nameStream = BehaviorSubject; integerStream = ReplaySubject)
-protocol RxVar {
-    var isEnabled: Observable<Bool> { get }
-    var nameStream: Observable<[EMobilitySearchVehicle]> { get }
-    var integerStream: Observable<Int> { get }
-}
-"""
-
-let rxObservablesMock = """
-
-class RxVarMock: RxVar {
-    init() { }
-    init(isEnabled: Observable<Bool> = PublishSubject<Bool>(), nameStream: Observable<[EMobilitySearchVehicle]> = BehaviorSubject<[EMobilitySearchVehicle]>(value: [EMobilitySearchVehicle]()), integerStream: Observable<Int> = ReplaySubject<Int>.create(bufferSize: 1)) {
-        self.isEnabled = isEnabled
-        self.nameStream = nameStream
-        self.integerStream = integerStream
-    }
-    private(set) var isEnabledSubjectSetCallCount = 0
-    var _isEnabled: Observable<Bool>? { didSet { isEnabledSubjectSetCallCount += 1 } }
-    var isEnabledSubject = PublishSubject<Bool>() { didSet { isEnabledSubjectSetCallCount += 1 } }
-    var isEnabled: Observable<Bool> {
-        get { return _isEnabled ?? isEnabledSubject }
-        set { if let val = newValue as? PublishSubject<Bool> { isEnabledSubject = val } else { _isEnabled = newValue } }
-    }
-    private(set) var nameStreamSubjectSetCallCount = 0
-    var _nameStream: Observable<[EMobilitySearchVehicle]>? { didSet { nameStreamSubjectSetCallCount += 1 } }
-    var nameStreamSubject = BehaviorSubject<[EMobilitySearchVehicle]>(value: [EMobilitySearchVehicle]()) { didSet { nameStreamSubjectSetCallCount += 1 } }
-    var nameStream: Observable<[EMobilitySearchVehicle]> {
-        get { return _nameStream ?? nameStreamSubject }
-        set { if let val = newValue as? BehaviorSubject<[EMobilitySearchVehicle]> { nameStreamSubject = val } else { _nameStream = newValue } }
-    }
-    private(set) var integerStreamSubjectSetCallCount = 0
-    var _integerStream: Observable<Int>? { didSet { integerStreamSubjectSetCallCount += 1 } }
-    var integerStreamSubject = ReplaySubject<Int>.create(bufferSize: 1) { didSet { integerStreamSubjectSetCallCount += 1 } }
-    var integerStream: Observable<Int> {
-        get { return _integerStream ?? integerStreamSubject }
-        set { if let val = newValue as? ReplaySubject<Int> { integerStreamSubject = val } else { _integerStream = newValue } }
-    }
-}
 
+    // Need to use the same name as the parent class to avoid name collision
+    typealias BarMock = parent.BarMock
```

Comment:
This typealias is workaround after moving to strict testing. I think this is okay.